<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>颜色</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_Color_Selector">
             <property name="title">
              <string>颜色选择</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QRadioButton" name="radioButton_red">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    background-color: #ff0000;
    color: white;
    border: 2px solid #880000;
    border-radius: 6px;
    padding: 8px;
    font-weight: bold;
}
QRadioButton:checked {
    border: 4px solid black;
}
QRadioButton::indicator {
    width: 0;
    height: 0;
}</string>
                </property>
                <property name="text">
                 <string>红</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="9">
               <widget class="QRadioButton" name="radioButton_purple">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    background-color: #ff00ff;
    color: white;
    border: 2px solid #550077;
    border-radius: 6px;
    padding: 8px;
    font-weight: bold;
}
QRadioButton:checked {
    border: 4px solid black;
}
QRadioButton::indicator {
    width: 0;
    height: 0;
}
</string>
                </property>
                <property name="text">
                 <string>紫</string>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QRadioButton" name="radioButton_yellow">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    background-color: #ffff00;
    color: black;
    border: 2px solid #aaaa00;
    border-radius: 6px;
    padding: 8px;
    font-weight: bold;
}
QRadioButton:checked {
    border: 4px solid black;
}
QRadioButton::indicator {
    width: 0;
    height: 0;
}
</string>
                </property>
                <property name="text">
                 <string>黄</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QRadioButton" name="radioButton_white">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    background-color: #ffffff;
    color: black;
    border: 2px solid #aa5500;
    border-radius: 6px;
    padding: 8px;
    font-weight: bold;
}
QRadioButton:checked {
    border: 4px solid black;
}
QRadioButton::indicator {
    width: 0;
    height: 0;
}
</string>
                </property>
                <property name="text">
                 <string>白</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="radioButton_green">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    background-color: #00ff00;
    color: white;
    border: 2px solid #006600;
    border-radius: 6px;
    padding: 8px;
    font-weight: bold;
}
QRadioButton:checked {
    border: 4px solid black;
}
QRadioButton::indicator {
    width: 0;
    height: 0;
}
</string>
                </property>
                <property name="text">
                 <string>绿</string>
                </property>
               </widget>
              </item>
              <item row="0" column="7">
               <widget class="QRadioButton" name="radioButton_cyan">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    background-color: #00ffff;
    color: black;
    border: 2px solid #009999;
    border-radius: 6px;
    padding: 8px;
    font-weight: bold;
}
QRadioButton:checked {
    border: 4px solid black;
}
QRadioButton::indicator {
    width: 0;
    height: 0;
}
</string>
                </property>
                <property name="text">
                 <string>青</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QRadioButton" name="radioButton_blue">
                <property name="styleSheet">
                 <string notr="true">QRadioButton {
    background-color: #0000ff;
    color: white;
    border: 2px solid #000088;
    border-radius: 6px;
    padding: 8px;
    font-weight: bold;
}
QRadioButton:checked {
    border: 4px solid black;
}
QRadioButton::indicator {
    width: 0;
    height: 0;
}
</string>
                </property>
                <property name="text">
                 <string>蓝</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="3" column="3">
               <widget class="QPushButton" name="pushButton_color_change_fix">
                <property name="text">
                 <string>手动</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QPushButton" name="pushButton_color_change_auto">
                <property name="text">
                 <string>自动</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>速度</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSlider" name="horizontalSlider_speed">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>灰度</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSlider" name="horizontalSlider_brightness">
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>255</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QSpinBox" name="spinBox_brightness">
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>255</number>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="spinBox_speed"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>渐变</string>
          </attribute>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>60</y>
             <width>120</width>
             <height>80</height>
            </rect>
           </property>
          </widget>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>60</y>
             <width>120</width>
             <height>80</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>老化测试</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QWidget" name="widget_7" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCheckBox" name="checkBox_aging_red">
                <property name="styleSheet">
                 <string notr="true">QCheckBox {
    background-color: red;
}
</string>
                </property>
                <property name="text">
                 <string>红</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_aging_green">
                <property name="styleSheet">
                 <string notr="true">QCheckBox {
    background-color: green;
}
</string>
                </property>
                <property name="text">
                 <string>绿</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_aging_blue">
                <property name="styleSheet">
                 <string notr="true">QCheckBox {
    background-color: blue
;
}
</string>
                </property>
                <property name="text">
                 <string>蓝</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_white">
                <property name="styleSheet">
                 <string notr="true">QCheckBox {
    background-color: white;
}
</string>
                </property>
                <property name="text">
                 <string>白</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_yellow">
                <property name="styleSheet">
                 <string notr="true">QCheckBox {
    background-color: yellow;
}
</string>
                </property>
                <property name="text">
                 <string>黄</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_purple">
                <property name="styleSheet">
                 <string notr="true">QCheckBox {
    background-color: magenta;
}
</string>
                </property>
                <property name="text">
                 <string>紫</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_cyan">
                <property name="styleSheet">
                 <string notr="true">QCheckBox {
    background-color: cyan;
}
</string>
                </property>
                <property name="text">
                 <string>青</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_9" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QWidget" name="widget_8" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QCheckBox" name="checkBox_HorizontalLine">
                   <property name="text">
                    <string>横线</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_VerticalLine">
                   <property name="text">
                    <string>竖线</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_DiagonalLeftLine">
                   <property name="text">
                    <string>左斜线</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBox_DiagonalRightLine">
                   <property name="text">
                    <string>右斜线</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_11" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="layoutDirection">
                    <enum>Qt::LayoutDirection::RightToLeft</enum>
                   </property>
                   <property name="text">
                    <string>间距</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBox_LineSpacing">
                   <property name="value">
                    <number>30</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_10" native="true">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="3" column="1">
               <widget class="QSlider" name="horizontalSlider_aging_brightness">
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>255</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QSpinBox" name="spinBox_speed_2"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>灰度</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>速度</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QSpinBox" name="spinBox_brightness_2">
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>255</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSlider" name="horizontalSlider_aging_speed">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>定位</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>扫描</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="6" column="5">
           <widget class="QSpinBox" name="spinBox_hight">
            <property name="maximum">
             <number>2160</number>
            </property>
           </widget>
          </item>
          <item row="6" column="4">
           <widget class="QLabel" name="label_hight">
            <property name="text">
             <string>高度</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QSpinBox" name="spinBox_y_value">
            <property name="minimum">
             <number>-1440</number>
            </property>
            <property name="maximum">
             <number>1440</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="label_window_pos_y">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="5">
           <widget class="QSpinBox" name="spinBox_width">
            <property name="maximum">
             <number>1440</number>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QCheckBox" name="checkBox_lock">
            <property name="text">
             <string>锁定</string>
            </property>
           </widget>
          </item>
          <item row="6" column="8">
           <widget class="QCheckBox" name="checkBox_windowTopMost">
            <property name="text">
             <string>窗口置顶</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_width">
            <property name="text">
             <string>宽度</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="spinBox_x_value">
            <property name="minimum">
             <number>-2160</number>
            </property>
            <property name="maximum">
             <number>2160</number>
            </property>
           </widget>
          </item>
          <item row="6" column="10">
           <widget class="QCheckBox" name="checkBox_double_fullScreen">
            <property name="text">
             <string>双击第二屏</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_window_pos_x">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="10">
           <widget class="QCheckBox" name="checkBox_hide">
            <property name="text">
             <string>隐藏</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="11">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_language">
         <property name="text">
          <string>语言</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_hide_config">
         <property name="text">
          <string>隐藏窗口参数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_run_space">
         <property name="text">
          <string>运行</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLED_Display_Demo">
    <property name="title">
     <string>LED_Display_Demo</string>
    </property>
   </widget>
   <addaction name="menuLED_Display_Demo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
